core/domain/beam_job_validators_test.py

```diff
@@ -0,0 +1,134 @@
+# coding: utf-8
+#
+# Copyright 2021 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Unit tests for core.domain.beam_job_validators_test."""
+
+from __future__ import absolute_import  # pylint: disable=import-only-modules
+from __future__ import unicode_literals  # pylint: disable=import-only-modules
+
+import ast
+
+from core.domain import prod_validation_jobs_one_off
+from core.domain import taskqueue_services
+from core.platform import models
+from core.tests import test_utils
+import python_utils
+
+(beam_job_models,) = models.Registry.import_models([models.NAMES.beam_job])
+datastore_services = models.Registry.import_datastore_services()
+
+
+class BeamJobValidatorTestBase(test_utils.AppEngineTestBase):
+    """Base class with helper methods for testing beam_job_validator jobs."""
+
+    AUTO_CREATE_DEFAULT_SUPERADMIN_USER = False
+
+    def run_job_and_get_output(self, entity_id_order=None):
+        """Runs the validation jobs and returns its output.
+
+        Args:
+            entity_id_order: list(str)|None. The ordering of IDs to be returned
```

Comment:
I don't understand this -- could you please clarify, in the docstring, why we need the argument?

If you're comparing list membership then it's better to just put things in sets and compare, in the test, that the sets are equal. Try to avoid introducing unnecessary additional logic into the tests.

Reply:
Obsolete, we don't actually need or use it so I've just deleted it.